desktop/cmp/dash/canvas/DashCanvasViewSpec.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2022 Extremely Heavy Industries Inc.
+ */
+import {DashViewSpec} from '@xh/hoist/desktop/cmp/dash/DashViewSpec';
+import '@xh/hoist/desktop/register';
+
+/**
+ * Spec used to generate DashCanvasViews and DashCanvasViewModels within a {@see DashCanvas}.
```

Comment:
Why, I do not know, but tsdoc does not consider `@see` to be an "inline" tag - it should be on its own line.  We can use `{@link FooBar}` instead for these common inline usages.